dvc/stage/params.py

```diff
@@ -11,3 +11,4 @@ class StageParams:
     PARAM_PARAMS = "params"
     PARAM_METRICS = "metrics"
     PARAM_PLOTS = "plots"
+    PARAM_DESC = "desc"
```

Comment:
We currently have PARAM_DESC in outs and in stages, because there is no general logic present for common fields. We'll be likely introducing them in the future with things like `if:`/etc, but for now keeping as is.